File: v2/snippets/testResources/ruby/TestClass.rb
module RubyTestClass
    class RubyTestClass
      @@static_value = 3
      @public_value
      @private_value

      def initialize(public_value, private_value)
        @public_value = public_value
        @private_value = private_value
      end

      def self.multiply_by_two(value)
        return value * 2
      end

      def get_public_value
        return @public_value
      end

      def set_public_value(public_value)
        @public_value = public_value
      end

      def get_private_value
        return @private_value
      end

      def set_private_value(private_value)
        @private_value = private_value
      end

      def multiply_two_numbers(first, second)
        return first*second
      end
    end
end

File: v2/snippets/testResources/python_package/TestClass.py

class PythonTestClass:

    static_value = 3

    def __init__(self, public_value, private_value):
        self.public_value = public_value
        self.private_value = private_value

    def get_public_value(self):
        return self.public_value

    def set_public_value(self, public_value):
        self.public_value = public_value

    def get_private_value(self):
        return self.private_value

    def set_private_value(self, private_value):
        self.private_value = private_value

    def multiply_two_numbers(self, first, second):
        return first*second

    @staticmethod
    def multiply_by_two(value):
        return value * 2

